ios/.gitignore

```diff
@@ -2,6 +2,7 @@ DerivedData
 .DS_Store
 project.xcworkspace
 xcuserdata
+build
 engine/KMEI/build
```

Comment:
These old build directories and products can be removed.

Reply:
Finally.  `engine/KMEI/build` took a fair bit of work today for this goal.